### ISSUE TITLE
The Android control bar is too ugly

### ISSUE THREAD

**Issue by herowws** [brianegan/chewie#601]
The Android control bar is too ugly
Can I customize the background color

**Comment by diegotori** [comment]
Yes, you can specify a `materialProgressColors` property pointing to a custom `ChewieProgressColors` in your `ChewieController`.
